apps/signal/51-function-call-effect/src/app/user.service.ts

```diff
@@ -1,10 +1,18 @@
-import { Injectable, signal } from '@angular/core';
+import { Injectable, signal, untracked } from '@angular/core';
 
 @Injectable({ providedIn: 'root' })
 export class UserService {
-  name = signal('Thomas');
+  name = untracked(signal('Thomas'));
```

Comment:
no, don't use it here. 
If you need to listen to name somewhere else in your application, you will not be able to.
If you want a basic property, just don't use a signal. 😅